workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
It's unclear to me why the variable `which` is needed, isn't it only used once on line 6?